shared/appengine.go

```diff
@@ -86,27 +86,31 @@ func (c *clientsImpl) Init(ctx context.Context) (err error) {
 // Close closes all clients in Clients. It must be called once and only once
 // before the server exits. Do not use AppEngineAPI afterwards.
 func (c *clientsImpl) Close() {
+	log.Printf("Closing clients")
 	if c.cloudtasks != nil {
-		if err := c.cloudtasks.Close(); err != nil {
+		client := c.cloudtasks
```

Comment:
May be worth a comment saying why we're doing this very weird thing. (And ditto below)

Reply:
Added a comment at the top of the function.